test/python/tests/rpc/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+#
+# {SRCDIR} should point to your working tree which should be your current pwd
+#
+
+version: '3'
+services:
+  ms1:
```

Comment:
Is there a specific format that we are following with the naming of test containers? I saw that some docker-compose files uses ms0 as the first contatiner name. 

Or is this done so that tests can be run parallely?

Reply:
Prior to this PR there has been a single docker-compose.yml file, and the tests have followed a tacit convention that containers are "numbered", starting from ms0. They also have fixed IP addresses where ms0 has address 10.0.0.2, ms1 has address 10.0.0.3, etc.

Since each (new) test now has its own custom docker-compose.yml, there is no reason why this convention *must* be followed. However, I would recommend that we continue do do so unless there is a good reason for doing otherwise. Having said that, there was one test that did not use "ms0" as its first container and, as I did not write it, decided to leave it as is - at least for now.

There is no intention at this time to run the pytests in parallel.